blocks/rich-text/format.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import { omitBy, map } from 'lodash';
+import { nodeListToReact } from 'dom-react';
+
+/**
+ * WordPress dependencies
+ */
+import { createElement, renderToString } from '@wordpress/element';
+
+/**
+ * Transforms a WP Element to its corresponding HTML string.
+ *
+ * @param {WPElement} value Element.
+ *
+ * @return {string}         HTML.
+ */
+export function elementToString( value ) {
+	return renderToString( value );
+}
+
+/**
+ * Transforms a string HTML to its corresponding WP element.
+ *
+ * @param {string}     value HTML.
+ *
+ * @return {WPElement}       Element.
+ */
+export function stringToElement( value ) {
```

Comment:
This function gives me bad vibes, for two reasons:

- The implementation is susceptible to XSS
   - As used, probably not in a way that would be of any practical concern
- We're performing two levels of transformation, first from string to DOM, then from DOM to element

Since its use is quite limited (for default value), maybe not of huge concern at the moment.

Reply:
Though it makes me wonder, why don't we just apply the default value as `dangerouslySetInnerHTML` ?

Reply:
Good idea it allows us to factorize more logic using `valueToString(value, format)`